Tests/GitHubReactions.Tests.ps1

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.Synopsis
+   Tests for GitHubReactions.ps1 module
+#>
+
+# This is common test code setup logic for all Pester test files
+$moduleRootPath = Split-Path -Path $PSScriptRoot -Parent
+. (Join-Path -Path $moduleRootPath -ChildPath 'Tests\Common.ps1')
+
+try
+{
+    # Define Script-scoped, readonly, hidden variables.
+    @{
+        defaultIssueTitle = "Test Title"
+        defaultCommentBody = "This is a test body."
+        defaultReactionType = "+1"
+        otherReactionType = "eyes"
+    }.GetEnumerator() | ForEach-Object {
+        Set-Variable -Force -Scope Script -Option ReadOnly -Visibility Private -Name $_.Key -Value $_.Value
+    }
+
+    Describe 'Creating, modifying and deleting comments' {
+        $repo = New-GitHubRepository -RepositoryName ([Guid]::NewGuid().Guid) -AutoInit
```

Comment:
While we're not Pester5 compliant yet, we're trying to get there.
Please don't put any setup code directly in `Describe`.  Move these two lines to a `BeforeAll` and the `Remove-GitHubRepository` to an `AfterAll`.